install_config/aggregate_logging.adoc

```diff
@@ -622,6 +622,31 @@ process with:
 
 ----
 $ oc get pods -w
+
+logging-curator-1541129400-l5h77           0/1       Running   0          11h  <1>
+logging-es-data-master-ecu30lr4-1-deploy   0/1       Running   0          11h  <2>
+logging-fluentd-2lgwn                      1/1       Running   0          11h  <3>
+logging-fluentd-lmvms                      1/1       Running   0          11h
+logging-fluentd-p9nd7                      1/1       Running   0          11h
+logging-kibana-1-zk94k                     2/2       Running   0          11h  <4>
+----
+
+<1> The Curator pod. Only one pod is needed for Curator.
+<2> The Elasticsearch pod on this host. 
```

Comment:
This is an es deploy pod, not es pod. The curator pod and the es pod in this example are not deployed successfully.  If you want to show the whole EFK stack here, it's better to show a healthy EFK stack, or it may cause misunderstanding.

Reply:
@QiaolingTang @VladMasarik  Do you have sample output that I can use.

Reply:
I believe this should be OK:
```
$ oc get pod
NAME                                      READY     STATUS    RESTARTS   AGE
logging-curator-1-gqbwl                   1/1       Running   0          2d
logging-es-data-master-soss6cj7-1-mvcfz   2/2       Running   0          2d
logging-fluentd-jwkb2                     1/1       Running   0          2d
logging-kibana-1-q5nkg                    2/2       Running   0          2d
```
although I am not sure if the number of kibana and ES pods is OK, but I think it is.

Reply:
@VladMasarik Thank you. I tried to stand up the EFK stack to get the proper output, but was unable to because of memory limitations on my servers.

Reply:
I would add to this as well that in 3.11, curator was moved to a BatchJob so you will only see a curator pod when the job is executing.  Otherwise, you will only see the others.